src/onegov/search/upgrade.py

```diff
@@ -2,3 +2,30 @@
 upgraded on the server. See :class:`onegov.core.upgrade.upgrade_task`.
 
 """
+
+from onegov.core.orm import Base
+from onegov.core.upgrade import upgrade_task
+from onegov.search.utils import searchable_sqlalchemy_models
+
+
+@upgrade_task('Adding full text search index column to postgres 42')
+def adding_full_text_search_columns_to_postgres(context):
+    # need to create all indexes in postgresql on every model in project
+    # for full text search. This will make elastic search setup obsolete.
+    # Ticket reference: ogc-508
+    #
+    # NOTE: This task can only be removed once all production systems got
+    # this upgrade
+    #
+    # onegov-core --select /onegov_org/risch upgrade
+    # onegov-core --select /onegov_org/* upgrade
+    # onegov-core --select /onegov_town6/meggen upgrade
+    # onegov-core --select /onegov_town6/* upgrade
+    # onegov-core upgrade
+
+    session = context.session
+    schema = context.schema
+
+    for model in searchable_sqlalchemy_models(Base):
```

Comment:
Shouldn't this be `Searchable` since `add_fts_column` is defined on that, and not on `Base`?

Reply:
Actually never mind. But you may need to also search for `libres.db.ORMBase`

Reply:
But both base classes Àllocation`and `Reservation`in libres do not define es_properties` as well as the two customization classes `CustomAllocation`and `CustomAllocation`. Shall I still add it to 'searchable' models?